api/search.go

```diff
@@ -375,6 +375,7 @@ func processCountQuery(ctx context.Context, elasticSearchClient DpElasticSearche
 		resCountChan <- nil
 		return
 	}
+	log.Info(ctx, "call count api with details", log.Data{"q": string(countQBytes)})
```

Comment:
This log will need to be removed later; would it be better to log the `countQbytes` if the query fails?